prov/efa/src/efa.h

```diff
@@ -426,6 +431,18 @@ bool efa_ep_support_rnr_retry_modify(struct fid_ep *ep_fid)
 #endif
 }
 
+/**
+ * @brief return weather this end point will write error cq entry for RNR
```

Comment:
endpoint should be one word, and "whether", not "weather"

Reply:
fixed in new revision